src/tests/JIT/Regression/JitBlue/Runtime_93876/Runtime_93876.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Optimize>True</Optimize>
+  </PropertyGroup>
+  <ItemGroup>
+    <CLRTestEnvironmentVariable Include="DOTNET_JitStressModeNames" Value="STRESS_MAKE_CSE STRESS_SPLIT_TREES_REMOVE_COMMAS" />
```

Comment:
In other tests there are comments that you need process isolation to use `CLRTestEnvironmentVariable`, is that no longer true?